partner_multi_relation_tabs/tests/test_relations.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests import common
+from lxml import etree
+
+
+class TestRelations(common.SingleTransactionCase):
+
+    post_install = True
+
+    def test_person_to_person_relation(self):
+        res_partner_relation_type_model = self.env['res.partner.relation.type']
+        res_partner_relation_type_selection_model = self.env[
+            'res.partner.relation.type.selection']
+        p2p_rel = res_partner_relation_type_model.create(
```

Comment:
Please for all create and write calls with multiple fields do:
```
some_model.create({
    'first_field': 'first_value',
    'second_field': 'second_value',
})
```
So put ( and { on one line,
then indent four spaces on all subsequent lines
then }) on last line.